DEVELOPMENT.md

```diff
@@ -186,7 +186,29 @@ or directly.
 
 A `.clang-format` file which defines our specific code style lives in the
 top level directory of the repository. The style is identical to the Google
-style.
+style. The Google c++ style guide can be found [here](https://google.github.io/styleguide/cppguide.html).
+It includes brief discussions or rationales for all the style decisions.
+
+Beyond what it is in the style guide we agreed to a few more additional rules
+specific to the Orbit project:
+
+### [[nodiscard]]
+We use "[[nodiscard]]" for all accessors of new classes. We do not touch
```

Comment:
In general use backticks instead of quotes so that markdown formats them as code.

Reply:
done.